src/java/org/apache/cassandra/service/accord/AccordKeyspace.java

```diff
@@ -191,6 +203,15 @@ public class AccordKeyspace
 
     private static final ClusteringIndexFilter FULL_PARTITION = new ClusteringIndexSliceFilter(Slices.ALL, false);
 
+    //TODO (now, performance): should this be partitioner rather than TableId?  As of this patch distributed tables should only have 1 partitioner...
+    private static final ConcurrentMap<TableId, AccordRoutingKeyByteSource.Serializer> TABLE_SERIALIZERS = new ConcurrentHashMap<>();
```

Comment:
this is primarily a read-cache, so it might be nice to optimise for this and not have the weight and cost of a concurrent hash map on read (e.g. CoW the whole map but use an Object2Object open hash map). This should also avoid infinite state growth under schema changes, so should either periodically be cleared, or should have entries removed when tables are dropped, or on update should clear any stale entries.